scripts/vm-util.py

```diff
@@ -277,6 +278,10 @@ def do_qemu(args):
         for device in args.usb_passthrough_pcap:
             qemu_args += qemu_process_usb_passthrough(device, True)
 
+    qemu_sudo = []
+    if args.sudo:
+        qemu_sudo = ["sudo"]
```

Comment:
launching graphical programs via sudo usually breaks, especially on wayland, and, more importantly, i'm not sure how good of an idea that is

Reply:
> launching graphical programs via sudo usually breaks, especially on wayland, and, more importantly, i'm not sure how good of an idea that is

Aren't there use cases for running QEMU with sudo, such as some networking configurations?

Reply:
I suspect it's never necessary - I'd need more details

Reply:
I doubt it was unavoidable.  we'd probably benefit more from supporting libvirt than running graphical programs with root, have you tried that?